### PR TITLE
Add needsRotation to user create, user verify

### PR DESCRIPTION
**Pull request by giarc3**
- `needsRotation` can be specified through `UserCreateOpts` when creating a user
- Default `needsRotation` is false
- `needsRotation` will be returned by user verify

### CODE REVIEW

File: src/lib.rs.in [hunk 1]
```diff
@@ -674,7 +688,7 @@ class IronSdk {
     self_type IronOxide;
     private constructor = empty;
     static_method user_verify(_:&str) -> Result<Option<UserVerifyResult>, String>; alias userVerify;
-    static_method user_create(_:&str, _:&str) -> Result<UserCreateKeyPair, String>; alias userCreate;
+    static_method user_create(_:&str, _:&str, _:&UserCreateOpts) -> Result<UserCreateKeyPair, String>; alias userCreate;
```
Comment (coltfred) [inline src/lib.rs.in]: It would be nice if the params had names and we had nice comments on these 2 like we do on other methods.

Comment (clintfred) [inline src/lib.rs.in]: Are we making sure to name params now, or no? I thought the latest rust_swig used them in the generated code?

File: Cargo.toml [hunk 2]
```diff
@@ -12,7 +12,7 @@ crate-type = ["cdylib"]
 [dependencies]
 log = "0.4"
 itertools = "0.8"
-ironoxide = "0.10.1"
+ironoxide = { git = "https://github.com/IronCoreLabs/ironoxide.git", branch = "17-changes-for-scala"}
```
Comment (clintfred) [inline Cargo.toml]: don't forget to depend on a released version

File: src/lib.rs.in (hunk 1 above)
Comment (giarc3) [reply src/lib.rs.in]: I named the parameters and brought the comments from ironoxide here as well

Comment (giarc3) [reply src/lib.rs.in]: Yes, I have now named the parameters and brought the documentation from ironoxide over

File: Cargo.toml (hunk 2 above)
Comment (giarc3) [reply Cargo.toml]: Done

**Pull request merged**